### PR TITLE
hey try this i dont kno how to complile what this is my first time sorry

### PR DESCRIPTION
**Pull request by bellol**
hey try this i dont kno how to complile what this is my first time sorry???????

2wilhelmek @wilhelmeek 22@222@2@EEK @WILLIAMEEK CAN U CHEck?????????????????

### CODE REVIEW

**Comment by bellol** [comment]
this is a good code

**Comment by bellol** [comment]
hey can merge???

**Comment by bellol** [comment]
quick i need to turn on my lights

**Comment by bellol** [comment]
haha sorry my first time writing java

**Comment by bellol** [comment]
hey try this i dont kno how to complile what this is my first time sorry???????

2wilhelmek @wilhelmeek 22@222@2@EEK @williameek CAN U CHEck?????????????????

**Comment by bellol** [comment]
hey try this i dont kno how to complile what this is my first time sorry???????

2wilhelmek @wilhelmeek 22@222@2@EEK @williameek CAN U CHEck?????????????????

**Comment by bellol** [comment]
sorry doble post

**Comment by bellol** [comment]
my internet is wireless

**Comment by wilhelmeek** [comment]
yes this is a a good code

**Comment by bellol** [comment]
i think u spell who wrong
rename who-claipper?

**Comment by wilhelmeek** [comment]
pls respond

**Comment by bellol** [comment]
Mi bombilla es muy caliente
Necesito apagarlo rápidamente
!!!!!!!!!!!!!!!!!! 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡

File: app.js [hunk 1]
```diff
@@ -5,21 +5,26 @@ const config = require('config.json')('./config.json');
...
 
 const clapConfig = {
-  AUDIO_SOURCE: config.audioSource,
-  CLAP_AMPLITUDE_THRESHOLD: config.threshold,
+	AUDIO_SOURCE: config.audioSource,
+	CLAP_AMPLITUDE_THRESHOLD: config.threshold,
 };
 
-let state = false;
+let isCurrentlyOn = false;
```
Comment (ramshaw888) [inline app.js:12]: Jesus fucking christ

File: app.js [hunk 2]
```diff
@@ -5,21 +5,26 @@ const config = require('config.json')('./config.json');
...
 
 const philipsHue = new PhilipsHue(config.bridge, config.username);
 philipsHue.initialiseLights(beginClap);
 
 function beginClap() {
-  clapDetector.start(clapConfig);
-  clapDetector.onClap(toggleLights.bind(this));
+	clapDetector.start(clapConfig);
```
Comment (ramshaw888) [inline app.js:18]: Pls no

File: PhilipsHue.js [hunk 3]
```diff
@@ -9,49 +9,65 @@ class PhilipsHue {
...
+      }
       console.log('Initialisation complete');
       done();
     });
   }
 
-  lightsList() {
-    return this.lights;
+  turnOnLights() {
+    console.log('Turning on lights ' + this.lights);
```
Comment (ramshaw888) [inline PhilipsHue.js:62]: Have you even run the linter?????

File: PhilipsHue.js [hunk 4]
```diff
@@ -9,49 +9,65 @@ class PhilipsHue {
     this.lights = [];
   }
 
+  function Light(id) {
+    this.id = id;
+    const url = `${this.bridgeIP}/api${this.username}/lights/${this.id}/state`;
+
+    function changeStateRequest(requestBody) {
```
Comment (ramshaw888) [inline PhilipsHue.js:16]: Let's not do this.

**Review by ramshaw888 — CHANGES_REQUESTED** [review]
Thanks for the contribution. Could you please include a summary of reasons to make these changes?

File: PhilipsHue.js (hunk 3 above)
Comment (bellol) [reply PhilipsHue.js:62]: hue is linter?? ur friend?????? u can tag him if u want up 2 u

**Comment by bellol** [comment]
i want to make changes to turn on my ligght!!!

**Pull request closed without merging**